docs/6weeks/redis-lock-patterns.md

```
@@ -0,0 +1,1237 @@
+  async unlock(key: string): Promise<boolean> {
+    const result = await this.redis.del(key);
+    return result === 1;
+  }
+
+  async isLocked(key: string): Promise<boolean> {
+    const exists = await this.redis.exists(key);
+    return exists === 1;
+  }
+
+  async extendLock(key: string, additionalSeconds: number): Promise<boolean> {
+    // 락이 존재할 때만 연장
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)